AutoscoperM/AutoscoperM.py

```diff
@@ -757,30 +767,29 @@ def onLoadPV(self):
             transformSubDir = self.ui.tfmSubDir.text
 
             vols = glob.glob(os.path.join(mainOutputDir, volumeSubDir, "*.tif"))
-            tfms = glob.glob(os.path.join(mainOutputDir, transformSubDir, "*.tfm"))
+            tfms_t = glob.glob(os.path.join(mainOutputDir, transformSubDir, "*_t.tfm"))
```

Comment:
Independently of this pull request, we should consider documentating the different transform files generated by Autoscoper/AutoscoperM.

May be by updating the following section: https://autoscoper.readthedocs.io/en/latest/file-specifications/index.html